icons/between-horizontal-end.json

```
@@ -33,6 +33,10 @@
     "account"
   ],
   "aliases": [
-    "between-horizonal-end"
+    {
+      "name": "between-horizonal-end",
+      "deprecated": true,
+      "deprecationReason": "Renamed because of typo. This export will be removed in v1.0"
```

I think adding this here feels wrong, I would prefer adding this in the generator.
I would vote to keep them after v1 in general, too much hassle for the user for not that much upside.
```suggestion
      "deprecationReason": "Renamed because of typo."
```

---

Ideally, after v1.0 deprecated aliases could be kept and automatically removed in the next major release following the one they were added in. If and when we get there I agree that the generator should add this information. For now I think this "dirty hack" should do, as the aforementioned functionality is just too much of a hassle.